app/views/configure/reading.phtml

```diff
@@ -68,6 +68,27 @@
 				</div>
 			</div>
 
+			<div class="form-group">
+				<label class="group-name" for="wrapTitle">Wrap long titles in Normal View and Global view</label>
```

Comment:
Something like clip or cut off, not wrap.

Reply:
Hm, although on second thought it'll wrap too… but wrap is a bit technical.

Use multiple lines for… ?

Reply:
Would it be better just to have `titles in Normal View and Global view`?

Then we need a good word for "no wrap" as first select list item

Reply:
I'd only looked at the CSS in detail; I misinterpreted "no wrapping" to mean "no clipping". One line doesn't make a lick of sense to me, hence the misunderstanding.

Reply:
I am not sure if I understand you correct.
Could you please give your idea of the left text (`wrap long titles in Normal View and Global view`) and of the first select option (`no wrap`)?